app/javascript/styles/mastodon/widgets.scss

```diff
@@ -93,12 +93,6 @@
       display: flex;
       align-items: center;
     }
-
-    .account__avatar {
-      width: 44px;
-      height: 44px;
-      background-size: 44px 44px;
```

Comment:
All I can see anywhere else is 36px, does this mean a design change?

Reply:
No, there was no place where these classes were used without overriding all of this to 36px